ls8/server/api/auth/auth.controller.js

```diff
@@ -0,0 +1,22 @@
+const UsersModel = require("../models/users.model");
+const { compareSync } = require("bcryptjs");
+
+exports.authUser = async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+    const user = await UsersModel.findOne({ email }).lean().exec();
+    if (!user) {
+      throw { status: 404, error: "user not exists" };
+    }
+
+    if (!compareSync(password, user.password)) {
+      throw { status: 403, error: "user not exists" };
```

Comment:
```suggestion
      throw { status: 404, error: "user not exists" };
```
Не надо выдавать что пароль неверный. Я так делал в лекции, чтобы легко было отдебажить, на проекте так нельзя.